ts/src/gate.ts

```diff
@@ -2141,10 +2145,21 @@ export default class gate extends Exchange {
             'margin': 'publicSpotGetTickers',
             'swap': 'publicFuturesGetSettleTickers',
             'future': 'publicDeliveryGetSettleTickers',
+            'option': 'publicOptionsGetTickers',
         });
+        if (market['type'] === 'option') {
+            const marketId = market['id'];
+            const optionParts = marketId.split ('-');
+            request['underlying'] = this.safeString (optionParts, 0);
+        }
         const response = await this[method] (this.extend (request, query));
-        const ticker = this.safeValue (response, 0);
-        return this.parseTicker (ticker, market);
+        const result = [];
+        for (let i = 0; i < response.length; i++) {
```

Comment:
@Dan-krm fethTicker should return a `Ticker`, but doing this we're changing the return type 
![image](https://github.com/ccxt/ccxt/assets/43336371/65438f06-f7a5-44d8-abb3-d815560b5e2c)

Reply:
Fair enough I knew there were some issues with my approach here, I like what you did instead